config/install/field.field.paragraph.oe_facts_figures.field_oe_paragraphs.yml

```diff
@@ -6,42 +6,83 @@ dependencies:
     - paragraphs.paragraphs_type.oe_fact
     - paragraphs.paragraphs_type.oe_facts_figures
   module:
+    - composite_reference
     - entity_reference_revisions
+third_party_settings:
+  composite_reference:
+    composite: false
+    composite_revisions: false
 id: paragraph.oe_facts_figures.field_oe_paragraphs
 field_name: field_oe_paragraphs
 entity_type: paragraph
 bundle: oe_facts_figures
 label: Fact
-description: 'Add between 1 to 6 Facts.'
+description: ''
 required: true
 translatable: false
 default_value: {  }
 default_value_callback: ''
 settings:
   handler: 'default:paragraph'
   handler_settings:
-    negate: 0
     target_bundles:
       oe_fact: oe_fact
+    negate: 0
     target_bundles_drag_drop:
-      oe_facts_figures:
-        weight: 14
-        enabled: false
       oe_accordion:
         weight: 15
         enabled: false
       oe_accordion_item:
         weight: 16
         enabled: false
+      oe_banner:
```

Comment:
where are all these aditiones  coming from ?

Reply:
same here, from re-saving the form.

Reply:
No, the demo module in the runner enables all submodules and that polutes your export, you need to disable that from the runner and enable only oe_paragraphs, then make the change and export only the change (ignore the other stuff)
```
diff --git a/config/install/field.field.paragraph.oe_facts_figures.field_oe_paragraphs.yml b/config/install/field.field.paragraph.oe_facts_figures.field_oe_paragraphs.yml
index 4a69c65..2bd6fc8 100644
--- a/config/install/field.field.paragraph.oe_facts_figures.field_oe_paragraphs.yml
+++ b/config/install/field.field.paragraph.oe_facts_figures.field_oe_paragraphs.yml
@@ -12,7 +12,7 @@ field_name: field_oe_paragraphs
 entity_type: paragraph
 bundle: oe_facts_figures
 label: Fact
-description: 'Add between 1 to 6 Facts.'
+description: ''
 required: true
 translatable: false
 default_value: {  }
@@ -20,13 +20,10 @@ default_value_callback: ''
 settings:
   handler: 'default:paragraph'
   handler_settings:
-    negate: 0
     target_bundles:
       oe_fact: oe_fact
+    negate: 0
     target_bundles_drag_drop:
-      oe_facts_figures:
-        weight: 14
-        enabled: false
       oe_accordion:
         weight: 15
         enabled: false
@@ -40,8 +37,11 @@ settings:
         weight: 18
         enabled: false
       oe_fact:
-        enabled: true
         weight: 19
+        enabled: true
+      oe_facts_figures:
+        weight: 14
+        enabled: false
       oe_links_block:
         weight: 20
         enabled: false
@@ -60,7 +60,4 @@ settings:
       oe_social_media_follow:
         weight: 25
         enabled: false
-      oe_text_feature_media:
-        weight: 26
-        enabled: false
 field_type: entity_reference_revisions
```